reference/calicoctl/ipam/check.md

```diff
@@ -0,0 +1,61 @@
+---
+title: calicoctl ipam check
+description: Command to check IPAM status
+canonical_url: '/reference/calicoctl/ipam/check'
+---
+
+This section describes the `calicoctl ipam check` command.
+
+Read the [calicoctl overview]({{ site.baseurl }}/reference/calicoctl/overview) for a full list of calicoctl commands.
+
+## Displaying the help text for 'calicoctl ipam check' command
+
+Run `calicoctl ipam check --help` to display the following help menu for the command.
+
+```
+Usage:
+  calicoctl ipam check [--config=<CONFIG>] [--show-all-ips] [--show-problem-ips] [-o <FILE>]
+
+Options:
+  -h --help                Show this screen.
+  -o --output=<FILE>       Path to output report file.
+  -c --config=<CONFIG>     Path to the file containing connection configuration in
+                           YAML or JSON format.
+                           [default: /etc/calico/calicoctl.cfg]
```

Comment:
Should we add help text for what `--show-all-ips` and `--show-problem-ips` is supposed to do?